gcloud-java-storage/src/main/java/com/google/gcloud/storage/BatchResponse.java

```diff
@@ -121,13 +121,16 @@ public String toString() {
   }
 
   @Override
-  public int hashCode() {
+  public final int hashCode() {
     return Objects.hash(deleteResult, updateResult, getResult);
   }
 
   @Override
-  public boolean equals(Object obj) {
-    if (!(obj instanceof BatchResponse)) {
+  public final boolean equals(Object obj) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary final modifier in final class](https://www.codacy.com/app/mziccard/gcloud-java/file/2392420400/issues/source?bid=3123103&fileBranchId=3228372#l129)